_posts/2021-02-08-infinispan-kubectl.adoc

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog
+title: Infinispan Kubectl
+permalink: /blog/:year/:month/:day/infinispan-kubectl
+date: '2021-02-08T12:00:00.000-00:00'
+author: remerson
+tags: [ "cli", "quarkus", "native", "kubernetes"]
+---
+
+Now that the Infinispan CLI is available as a TODO[native binary] it's possible to utilise it as a kubectl client plugin.
+
+Integrating with `kubectl` lets you install and remove the Infinispan Operator, create and delete Infinispan clusters, and obtain information about various resources.
+
+== Installation
+To install the CLI as a `kubectl` plugin, do the following:
+
+. Copy the `ispn-cli`, or create a hard link, to a file named "kubectl-infinispan".
```

Comment:
@ryanemerson hard link doesn't work for 12.1.0.Final, but works for 12.0.0.Final and vise versa 
![image](https://user-images.githubusercontent.com/24249373/114342296-4913c780-9b64-11eb-91c4-386194c458df.png)